OVRLighthouseManager/Services/LighthouseGattService.cs

```diff
@@ -52,58 +53,34 @@ public async Task SleepAsync(Lighthouse lighthouse)
 
     public async Task StandbyAsync(Lighthouse lighthouse)
     {
-        if (lighthouse.Version == LighthouseVersion.V1)
+        if ( lighthouse.Version == LighthouseVersion.V1 )
         {
-            throw new LighthouseGattException("Standby is not supported on V1 lighthouses");
-        }
-        await WriteV2PowerCharacteristic(lighthouse, 0x02);
-    }
-
-    private async Task ControlV1Async(Lighthouse lighthouse, bool powerOn)
-    {
-        byte[] bytes;
-        if (powerOn)
-        {
-            bytes = new byte[] { 0x12, 0x00, 0x00, 0x00 };
+            await WriteV1PowerCharacteristic(lighthouse, 0x01, 0x04);
         }
         else
         {
-            bytes = new byte[] { 0x12, 0x02, 0x00, 0x01 };
-        }
-
-        if (string.IsNullOrEmpty(lighthouse.Id))
-        {
-            throw new LighthouseGattException("ID is missing in lighthouse settings");
+            await WriteV2PowerCharacteristic(lighthouse, 0x02);
         }
-        if (lighthouse.Id.Length != 8)
-        {
-            throw new LighthouseGattException($"Invalid ID length: {lighthouse.Id} ({lighthouse.Id.Length})");
-        }
-
-        // convert id to byte array
-        var idBytes = Enumerable.Range(0, lighthouse.Id.Length)
-            .Where(x => x % 2 == 0)
-            .Select(x => Convert.ToByte(lighthouse.Id.Substring(x, 2), 16));
-
-        bytes = bytes
-            .Concat(idBytes.Reverse())
-            .Concat(Enumerable.Repeat<byte>(0x00, 12))
-            .ToArray();
-
-        if (bytes.Length != 20)
-        {
-            throw new LighthouseGattException("Invalid byte array length");
-        }
-
-        await WriteV1PowerCharacteristic(lighthouse, bytes);
     }
 
-    private async Task WriteV1PowerCharacteristic(Lighthouse lighthouse, byte[] data)
+    private async Task WriteV1PowerCharacteristic(Lighthouse lighthouse, byte cmd, byte val)
     {
-        await WritePowerCharacteristicAsync(lighthouse, V1ControlService, V1PowerCharacteristic, data);
+        // Lighthouse 1.0 commands are of the following form:
+        // 0x12 : HTC MAGIC
+        // command ::
+        //     0x00 : Power On
+        //     0x02 : Sleep
+        //     0x01 : Standby

```

Comment:
Standby command can turn off motors and LEDs after 1 minutes even if I set timeout param to 0. I think it's too late.